profiles/student/axk4545.yml

```diff
@@ -0,0 +1,12 @@
+name: Aidan Kahrs
+website: https://axk
+blog: https://axk4545.me
+email: public@axk4545.me
+major: Networking and Systems Administration
+graduation: 2018
+handle: axk4545
+forges:
+  GitHub: https://github.com/axk4545
+  GitLab: https://gitlab.com/axk4545/
```

Comment:
Nice idea with GitLab. This can probably replace `BitBucket` in the CONTRIBUTING guidelines… I don't even know if BitBucket is around.

Reply:
BitBucket is still around